### DISCUSSION TITLE
Introducing File editing on GitHub Android! (public beta)

### DISCUSSION THREAD

**Discussion by alcere** [community/community#40852] [Mobile]
We're thrilled to announce the Beta launch of File editing on GitHub Android! You can now edit any open pull request on the go. We would love for you to try it out and give us your early feedback here - Do you find the feature useful? What else do you want us to support and why?

## How do I test this

1. If you are already a beta user - simply update to the latest beta build through Google Play Store. If you are not a beta user yet, join GitHub Mobile beta program through the following link: [Join Google Play Beta](https://play.google.com/apps/testing/com.github.android)
2. Open any existing pull request using the Android Beta GitHub app.
3. Click on **files changed** to review the changes.
4. Find a file you want to change, open the overflow menu from the file header, and press **Edit file**.

https://user-images.githubusercontent.com/18210473/205701098-9e155179-557a-4c47-942d-199920bede9d.mov

Happy testing! ❤️

**Comment by Mrxx99** [comment]
That is nice, but why only on Pull Requests and not under "Browse code" to also alow creating commits and a PR?

**Reply by alcere** [reply]
Hi @Mrxx99, we are happy to announce that editing code under "Browse code" and pull request creation is now available for Beta users!

**Reply by Mrxx99** [reply]
Works great, thanks for the notice!

**Reply by Mrxx99** [reply]
Is it in some way possible to edit multiple files that result in only one commit?

**Reply by alcere** [reply]
Sadly not at the moment

**Comment by nmitic** [comment]
This is great. Are there any plans to enable adding of files? We already have the same for the web based app. It would be great.

**Reply by alcere** [reply]
Hi thanks for the feedback @nmitic! Currently, there are no plans to enable adding new files, but we want to include it sooner or later!

**Comment by hasegawa0x3** [comment]
This is great. wonderful.

**Comment by Ronnie6464** [comment]
https://github.com/bitcoin-core-review-club/website/pull/649#discussion_r1171188763

**Reply by Ronnie6464** [reply]
[]()

**Comment by Mehrzad-SooteDelan** [comment]
A give Configouriton Of v2rayNg.

**Comment by DenisLisica** [comment]
> We're thrilled to announce the Beta launch of File editing on GitHub Android! You can now edit any open pull request on the go. We would love for you to try it out and give us your early feedback here - Do you find the feature useful? What else do you want us to support and why?
> 
> ## How do I test this
> 
> 1. If you are already a beta user - simply update to the latest beta build through Google Play Store. If you are not a beta user yet, join GitHub Mobile beta program through the following link: [Join Google Play Beta](https://play.google.com/apps/testing/com.github.android)
> 2. Open any existing pull request using the Android Beta GitHub app.
> 3. Click on **files changed** to review the changes.
> 4. Find a file you want to change, open the overflow menu from the file header, and press **Edit file**.
> 
> https://user-images.githubusercontent.com/18210473/205701098-9e155179-557a-4c47-942d-199920bede9d.mov
> 
> Happy testing! ❤️
>

**Reply by nmitic** [reply]
Awesome 👌 👏

**Reply by nmitic** [reply]
I find a very cool use case for it. 

I'm writing my blog posts using markdown files. 

Being able to update them and write new one using my phone is super cool. As I hooked everything up, so I can build the blog website using a commit. 

Now, what would be super cool is if we can use rich text editor while editing markdown files.

**Comment by akhilsahuji** [comment]
This is nice.
can we also add functionality of creating new files (we already have this in web version) and a editor like we have for creating issues ?
If yes , then GitHub mobile app will become my goto app for Blogging or note taking also.

**Reply by alcere** [reply]
Hi! thanks for the feedback @akhilsahuji ! Currently, there are no plans to enable adding new files, but we want to include it sooner or later!

**Comment by Zorbey99** [comment]
Check

**Comment by Giangplay** [comment]
How to create files:(

**Reply by alcere** [reply]
Hi @Giangplay! The latest Android and iOS update includes the option to create files. To create a new file, just navigate to the Repository where you want to add a new file and tap on the "Create file" option located in the top bar overflow menu (thee dots icon).

**Comment by sbe-arg** [comment]
Is this stale?
